### ISSUE TITLE
White border on Target Dropdown

### ISSUE THREAD

**Issue by achaido** [Open-Sustainability-Index/open-sustainability-index-frontend#28]

**Comment by achaido** [comment]
Change from dark to light / white border-color on Target dropdown + arrow. It's difficult to see it currently.

**Comment by ghritak** [comment]
Hi @achaido , Can i work in this issue?

**Comment by achaido** [comment]
Yes of course! Go ahead! On 7 Jun 2024, at 06:48, Ghritak Jyoti Kalita ***@***.***> wrote:﻿
Hi @achaido , Can i work in this issue?

**Comment by achaido** [comment]
Absolutely!

**Comment by hankmander** [comment]
Welcome @ghritak ! Do you need any assistance setting up your dev environment or did it work out of the box?

**Comment by ghritak** [comment]
Thanks, @hankmander ! I have set up both the frontend and backend repos. However, I'm currently busy and might not be able to start until after the 20th. I will need some help integrating the backend with the frontend when I begin. Is that okay?

**Comment by hankmander** [comment]
Yes, we'll do our best. Just reach out! @ghritak

**Comment by ghritak** [comment]
@tomsoderlund I have sent you a mail.
